packages/cubejs-backend-shared/src/env.ts

```diff
@@ -8,10 +8,6 @@ const variables = {
     .default(4000)
     .required()
     .asPortNumber(),
-  tlsPort: () => get('TLS_PORT')
-    .default(4433)
-    .required()
-    .asPortNumber(),
   tls: () => get('CUBEJS_ENABLE_TLS')
```

Comment:
Should this property also be removed?

Reply:
@hassankhan, I am still using it to detect that CUBEJS_ENABLE_TLS is used 😄

Reply:
Ahh yes, good point 😅